index.html

```diff
@@ -11,39 +11,212 @@
                     <li><a href="https://www.linkedin.com/in/muhammad-porag-nsu-cse/"><img src="./assets/social/Linkedin icon.svg" alt="Linkedin"></a></li>
                     <li><a href="#"><img src="./assets/social/medium.svg" alt="medium"></a></li>
                     <li><a href="https://github.com/porag-m06"><img src="./assets/social/github.svg" alt="github"></a></li>
                     <li><a href="#"><img src="./assets/social/angellist.svg" alt="angellist"></a></li>
                 </ul>
+                <a href="#" class="get-resume">Get my resume</a>
+
+                <div class="skillList">
+
+                    <div class="skillframe">
+                        <div class="lframe-head">
+                            <h3>Languages</h3>
+                            <img src="./assets/about/Dropdown_down.svg" alt="Languages Dropdown Btn">
+                        </div>
+                        <ul class="l-list ">
+                            <li class="lang"><img src="./assets/about/EllipseJS.svg" alt="JS"><h4>JavaScript</h4></li>
+                            <li class="lang"><img src="./assets/about/EllipseHTML.svg" alt="HTML"><h4>HTML</h4></li>
+                            <li class="lang"><img src="./assets/about/EllipseCSS.svg" alt="CSS"><h4>CSS</h4></li>
+                           
+                        </ul>
+                    </div>
+
+                    <div class="skillframe bdr-btm">
+                        <div class="lframe-head">
+                            <h3>Frameworks</h3>
+                            <img src="./assets/about/Dropdown_right.svg" alt="Languages Dropdown Btn">
+                        </div>
+                    </div>
+
+                    <div class="skillframe bdr-btm">
+                        <div class="lframe-head">
+                            <h3>Skills</h3>
+                            <img src="/assets/about/Dropdown_right.svg" alt="Languages Dropdown Btn">
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <secdition class="contact">
+                <div class="text-primary mod-text">
+                    <h1 class="primary-one zero center-white">Contact me</h1>
+                    <p class="contact-primary-two center-white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                        I’d love to help with it!</p>
+                </div>
+                <div class="from-container">
+                    <form>
+                        <input type="text" name="name" id="name" placeholder="Lorem Ipsum">
+                        <input type="email" name="email" id="emil" placeholder="loremipsum@gmail.com">
+                        <textarea id="message" name="message" placeholder="Write your message here..."></textarea>
+                    </form>
```

Comment:
- [ ] I notice that you implemented the contact form on this milestone of your project; it looks beautiful, but it was not required for this milestone; please remove this section as it is the next project. :wink: :wink:

Reply:
Sure. Thanks for the feedback.